authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+---
+parent:
+  title: Understanding the Authz Module
+order: 0
+description: Use the Cosmos SDK authz module to grant authorizations from one account (the granter) to another account (the grantee).
+---
+
+# Authz Module
+
+The authz ("authorization") module enables a granter to grant authorizations to a grantee that allows the grantee to execute messages on behalf of the granter.
+
+## Use Authz to Grant Authorizations
+
+By implementing the [authz module](https://docs.cosmos.network/v0.43/modules/authz/), Cosmos SDK application developers give users the ability to grant certain privileges to other users. For example, a user might want another user to vote on their behalf and so, rather than giving the other user access to their account, the user would grant an authorization that allows the other user to execute `MsgVote` on their behalf.
+
+How users decide to use the authz module is up to them. In one case, a validator might want to create a separate account for voting in order to keep their validator key more secure. In another case, a DAO might want to distribute authorizations to members of the DAO in which case a multisg account would grant authorizations to individual accounts and members of the DAO would be able to execute messages without requiring signatures from all the other members.
```

Comment:
since we haven't talked about DAO yet, let's define it
(did you know we are working on a Cosmos SDK-approved word list so we can all use consistent spelling and style for ecosystem terminology?) see https://docs.google.com/spreadsheets/d/15raX8lfzzeeifGrDiXSFTHaEDbsMVlFJzKgnNelihjs/edit#gid=0 for https://github.com/cosmos/advocates/issues/23

Reply:
I did not know. Thanks for sharing!